pkg/genericapiserver/genericapiserver.go

```diff
@@ -892,6 +894,25 @@ func (s *GenericAPIServer) InstallSwaggerAPI() {
 		},
 	}
 	swagger.RegisterSwaggerService(swaggerConfig, s.HandlerContainer)
+	openAPIConfig := openapi.Config{
```

Comment:
I understand now, @smarterclayton sorry for my misunderstanding. Yes the part of config that are specific to kubernetes need to be moved to genericapiserver's config and be set in master.